src/content/en/updates/2018/05/lighthouse.md

```diff
@@ -234,6 +234,21 @@ loading performance by converting their GIFs to video. Additionally, if your
 site is on a hosting plan with metered bandwidth, the potential cost savings
 alone should be enough to persuade you!
 
+## Give Lighthouse a try!
+
+If you're excited about these new audits, update Lighthouse and give them a try!
+
+* For Node users: Run `npm update lighthouse`, or `npm update lighthouse -g` if
+you installed Lighthouse globally.
+* For Chrome users:
+  * The Lighthouse extension should automatically update, but you can manually
+update it via `chrome://extensions`.
+  * In DevTools, The Audits panel will be shipping with `[version]` in Chrome
+`[version]`. You can check your version of Chrome via `chrome://version`. Chrome
```

Comment:
@kaycebasques I'd love for us to coordinate with the blog post you wrote in the LH 3.0 round-up. Should we say 3.0 in here for consistency? I was considering if we should say "Many of the audits we have covered in this post have been landing in Lighthouse over the last few versions. The very latest audits can be found in Lighthouse 3.0." or something to that effect.